app/mailers/notifications.rb

```diff
@@ -9,11 +9,27 @@ def import_approved(import)
       :subject => "Your Safecast import has been approved")
   end
 
+  def import_rejected(import)
+    @import = import
+    mail(
```

Comment:
Might be good to reformat these at some point. The indentation & parens isn't very idomatic. Nbd for this PR though.